content/documentation/overview/index.adoc

```diff
@@ -82,11 +82,11 @@ image::workload-view-pods.png[Demo Website]
 
 === Services
 
-In the service tab we go to the page detail of a service, we have a health icon to check if our service has some problem like error rate.
+In the Services tab we go to the page details of a service, we have a health icon to check if our service has some problem like error rate.
 
-In this page we have an overview about the service : labels, type, ip , ports, endpoints, workloads, destiantion rules, virtual services...
+In this page we have an overview about the service: labels, type, ip, ports, endpoints, workloads, destiantion rules, virtual services...
 
-We can also visualize the inbound/outbound metrics for this service and go to his grafana dashboard.
+We can also visualize the inbound/outbound metrics for this service and go to its Grafana dashboard.
```

Comment:
visualize -> maybe "view" or "display" sounds better?

Reply:
and same about "we can" here.
"Inbound/outbound metrics are displayed for this service and a more detailed view is available in its Grafana dashboard."